app/assets/javascripts/oxalis/controller/combinations/volumetracing_plane_controller.js

```diff
@@ -222,6 +225,20 @@ class VolumeTracingPlaneController extends PlaneControllerClass {
   getKeyboardControls(): Object {
     return _.extend(super.getKeyboardControls(), {
       c: () => Store.dispatch(createCellAction()),
+      "ctrl + i": async event => {
+        const mousePosition = Store.getState().temporaryConfiguration.mousePosition;
+        if (mousePosition) {
+          const [x, y] = mousePosition;
+          const globalMousePosition = calculateGlobalPos({ x, y });
+          const cube = Model.getSegmentationBinary().cube;
+          const mapping = event.altKey ? cube.mapping : null;
```

Comment:
I'm looking forward to further merge conflicts/errors :D Nothing you can do about that, though :)

Reply:
Yeah, I thought the same when I saw your PRs ^^